docs/pages/guides/adding-a-new-signer-to-identity-kit.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Adding a new signer to Identity Kit"
+date: "2024-08-05"
+authors:
+  - name: "Dmytro Sheremet"
+---
+
+# Adding a New Signer to Identity Kit
+
+To add your wallet as a signer to Identity Kit, follow these four steps:
+
+1. Clone the Identity Kit repository.
+2. Add your wallet configuration.
+3. Create a pull request.
+4. Wait for approval and then merge the pull request.
+
+## Steps
+
+### 1. Clone the Identity Kit repository
+
+First, clone the Identity Kit repository to your local machine. Open your terminal and run the
+following command:
+
+```bash
+git clone https://github.com/internet-identity-labs/identitykit.git
+```
+
+### 2. Add your wallet configuration
+
+Open the `packages/identitykit/src/lib/signers.ts` file in your preferred code editor and add your
+wallet configuration. Ensure that each field is correctly filled out:
+
+- id: A unique identifier for your wallet. Make sure it is unique to avoid conflicts.
+- providerUrl: The URL to connect to your wallet. This URL should be compatible with ICRC standards.
```

Comment:
let's add a link to [ICRC standards](https://github.com/dfinity/wg-identity-authentication/tree/main) here
https://github.com/dfinity/wg-identity-authentication/tree/main

Reply:
Fixed, thanks!